pkg/transferer/load.go

```diff
@@ -109,6 +113,13 @@ func (c *LoadChecker) updateStatus() {
 		status = LoadStatusWait
 		log.Warn().Err(err).Msgf("Error while checking metrics load")
 	}
+	if status == LoadStatusWait {
+		c.waitAttemptsCounter++
+		if c.waitAttemptsCounter > MaxWaitStatusAttempts {
+			log.Debug().Msgf("Reached max WAIT status attempts. Sending TERMINATE status")
```

Comment:
please, use constants for status keys

Reply:
Resolved in d2bdbaca363c55fe9de7934d01ac8667233954f2